backend/tests/models/test_mapbox.py

```diff
@@ -0,0 +1,42 @@
+"""Test MapBox API."""
+import os
+import requests
+
+from backend.models import distance
+
+import mock
+
+
+class MockResponse:
+    """Mock HTTP responses."""
+
+    def __init__(self, json_data, status_code):
+        self.json_data = json_data
+        self.status_code = status_code
+
+    def json(self):
+        return self.json_data
+
+    def raise_for_status(self):
+        if self.status_code != 200:
+            raise requests.exceptions.HTTPError(
+                '{}'.format(self.status_code),
+                response=self
+            )
+
+
+class TestMapBoxAPI():
+    """Test base metrics in the distance module."""
+    def setup(self):
+        """Initialize a measurer for use in the test cases."""
+        self.measurer = distance.MapBoxDrivingTime(api_key=os.environ.get('MAPBOX_TOKEN'))
+
+    @mock.patch('requests.get')
+    def test_matrix(self, mock_get):
+        """Check that the driving time matches expectations."""
+        json_data = {'durations': [[35799.4]]}
+        mock_get.return_value = MockResponse(json_data=json_data, status_code=200)
+        matrix_durations = self.measurer.get_matrix(
+            source_points=[(41.49008, -71.312796)], destination_points=[(41.499498, -81.695391)]
```

Comment:
Nit: Points should be longitude, latitude according to the function docstring.

Reply:
Updated doc, just want it to be consistent with the other functions